Model/Payment/Product.php

```diff
@@ -38,9 +44,18 @@ public function findOrCreateProducts($order)
         return $list;
     }
 
-    private function findOrCreateProduct($itemSku, $itemName)
+    private function getItemPrice($item, $productType)
     {
+        if ('bundle' == $productType)
+            return 0;
```

Comment:
pq o preço fica 0 se for bundle?

Reply:
Eu expliquei na Issue #42, mas o Bundle é um 'pacote de produtos', onde cada produto tem um valor e todos são somados para gerar o valor total.
O que acontecia era que o valor estava sendo enviado também no produto do tipo **bundle**.
Ex:
- Na loja é exibido um Pacote de balas (preço 10 reais)
- Na fatura é exibido:
  - Pacote de balas R$ 10,00 x1
  - bala de banana R$ 0,10 x100
**Total R$ 20,00**

- Onde o correto seria:
  - Pacote de balas R$ 0,00
  - bala de banana R$ 0,10 x100
**Total R$ 10,00**

## 
Isso se dá porque o pacote de balas não é um produto, e sim um pacote.

Reply:
:)